great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -340,6 +340,7 @@ def _get_bootstrap_confidence_interval(
                 vectorized=False,
                 confidence_level=confidence_level,
                 random_state=rng,
+                method="basic",
```

Comment:
Could you give me a quick explanation as to what this arg does? Does it help mitigate the `NaN` issue we were dealing with in any way?

Reply:
@cdkini Yes -- it exercises a simpler mode of the bootstrapping algorithm, which for our purposes yields the same results.  The other mode ("BCa" -- bias correction and acceleration -- was running into problems of getting NaNs for percentiles, and I contacted the author and submitted the bug report with tracing to him).  Once it is fixed, we can go back to it as well.